src/main/java/pl/futurecollars/invoicing/db/FileBasedDatabase.java

```diff
@@ -7,11 +7,13 @@
 import java.util.UUID;
 import java.util.stream.Collectors;
 import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Repository;
 import pl.futurecollars.invoicing.config.Configurations;
 import pl.futurecollars.invoicing.model.Invoice;
 import pl.futurecollars.invoicing.utils.FileService;
 import pl.futurecollars.invoicing.utils.JsonService;
 
+@Repository
```

Comment:
should be `@Service` or `@Component` - @repository is special kind of component reserved for managing DAO

Reply:
Done